src/components/baseComponents/ShortBrickDescription.scss

```diff
@@ -2,133 +2,105 @@
 @import '../../assets/sass/partials/mixins';
 @import '../../assets/sass/partials/extensions';
 
-.short-brick-info {
-  position: absolute;
-  top: 0.3vh;
-  left: 3vw;
-
-  .link-description {
-    text-transform: uppercase;
-    font-family: $font-family-bold;
-    line-height: 1.1;
-    margin-bottom: 0.5vh;
-    @include phone_and_tabletP {
-      overflow: hidden;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      font-size: 3.8vw;
-      width: 100%;
-    }
-    @include tabletP {
-      font-size: 0.8vw;
-      overflow: hidden;
-      width: 19vw;
-    }
-  }
-  .link-info {
-    color: $lighter-blue;
-    line-height: 1.3;
-    font-family: $font-family-regular;
-    overflow-y: hidden;
-    margin-bottom: 0.1vh;
-
-    overflow: hidden;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    @include phone_and_tabletP {
-      font-size: 3.8vw;
-      width: 75vw;
-    }
-    @include tabletP {
-      font-size: 0.9vw;
-      width: 19vw;
-    }
-  }
-}
-
```

Comment:
@farmanahmed2007  don`t understand why you moved short-brick-info class to sidebar.scss. Is is only used in ShortBrickDescription.tsx file. This is not related to sidebar. It is main content.
![image](https://user-images.githubusercontent.com/18567511/87118243-8bca0b80-c283-11ea-8204-80e078ac3f83.png)
Please move it  back to ShortBrickDescription.scss so it will be easier to understand in the future.